translations/messages.fr.xlf

```diff
@@ -1516,6 +1524,58 @@
       </trans-unit>
+      <trans-unit id="2724052802065983013" datatype="html">
+        <source>You cannot move the piece holding the ball.</source>
+        <target>Vous ne pouvez pas déplacer la pièce qui tient la balle.</target>
+      </trans-unit>
+      <trans-unit id="5364606037550605842" datatype="html">
+        <source>You must move by exactly one orthogonal space.</source>
+        <target>Vous devez déplacer la pièce d'exactement une case orthogonalement.</target>
+      </trans-unit>
+      <trans-unit id="2737840191812588251" datatype="html">
+        <source>The path of this pass is obstructed.</source>
+        <target>Le chemin de cette passe est obstrué.</target>
+      </trans-unit>
+      <trans-unit id="6962674839842537174" datatype="html">
+        <source>A pass must be done in a straight line, orthogonally or diagonally.</source>
+        <target>Une passe doit être faite le long d'une ligne droite, orthogonalement ou verticalement.</target>
+      </trans-unit>
+      <trans-unit id="7841459528155785625" datatype="html">
+        <source>You can only perform one pass per turn.</source>
+        <target>Vous ne pouvez faire qu'une seule passe par tour.</target>
+      </trans-unit>
+      <trans-unit id="898651979160418937" datatype="html">
+        <source>You can only perform two translations per turn, not three.</source>
+        <target>Vous ne pouvez faire que deux translations par tour, pas trois.</target>
+      </trans-unit>
+      <trans-unit id="6313782238959628837" datatype="html">
+        <source>The goal of Diaballik is to bring your ball, currently being held by your center piece as represented by two circles, into the home line of the opponent.</source>
+        <target>Le but à Diaballik est d'amener votre balle, initialement tenue par votre pièce centrale tel que représenté par le double cercle, du côté adverse.</target>
+      </trans-unit>
+      <trans-unit id="488799603315113699" datatype="html">
+        <source>Translations</source>
+        <target>Translations</target>
+      </trans-unit>
+      <trans-unit id="6723437388577428437" datatype="html">
+        <source>During a turn, you are allowed up to three actions, including at most two translations. A translation is an orthogonal step from any piece that does not hold the ball.&lt;br/&gt;&lt;br/&gt;Move your leftmost piece by one step. Once you are done, click on the green button that will appear on the bottom right of the board to indicate that you are done with your turn.</source>
+        <target>Durant un tour, vous êtes autorisés à faire jusque trois actions, qui incluent au plus deux translations. Une translations est un déplacement orthogonal d'une case d'une pièce qui ne tient pas la barre.&lt;br/&gt;&lt;br/&gt;Déplacez votre pièce la plus à gauche d'une case. Une fois que vous avez fait cela, cliquez sur le bouton vert qui apparaîtra en bas à drote du plateau, afin d'indiquer que vous avez fini votre tour.</target>
```

Comment:
LACHEZ LA BARRE !
Aussi "tenir la balle" ? C'pas fou comme formulation

Reply:
la balle qui est dans ta main, tu la tiens pas ? Tu la porte ? Tu es le détenteur de la balle ? "une pièce qui n'a pas la balle" me semble moins fou encore

Reply:
*of a piece.
Et va pour old balls !!